Drivers/Src/CANbus.c

```diff
@@ -16,22 +37,84 @@ void CANbus_Init(void) {
  * @param   CAN bus line
  * @param   id : CAN id of the message
  * @param 	payload : the data that will be sent.
+ * @param blocking: Whether or not this should be a blocking call
  * @return  0 if data wasn't sent, otherwise it was sent.
  */
-int CANbus_Send(CANId_t id, CANPayload_t payload) {
+ErrorStatus CANbus_Send(CANId_t id, CANPayload_t payload, CAN_blocking_t blocking) {
+    CPU_TS timestamp;
+    OS_ERR err;
+    //make sure that Can mailbox is available
+    if (blocking == CANBLOCKING){ 

```

Comment:
CANBLOCKING is not defined

Reply:
Same as the previous comment, it should be defined on line 43 of CANbus.h